server/pbenchinacan/etc/rsyslog.d/95-output-viaq-forwarding.conf

```diff
@@ -0,0 +1,59 @@
+# This is basically the RSYSLOG_SyslogProtocol23Format, which is RFC 5424 on
+# the wire, but with the message payload a CEE/Lumberjack JSON document.
+template(
+    name="ViaQ_SyslogProtocol23Format"
+    type="string"
+    string="<%PRI%>1 %TIMESTAMP:::date-rfc3339% %HOSTNAME% %APP-NAME% %PROCID% %MSGID% %STRUCTURED-DATA% @cee:%$!%\n"
+)
+
+# Some log messages have not been stored in the JSON variables, so check and
+# add them here.
+if strlen($!msg) < 1 then {
+    set $!msg = $msg;
+}
+
+# Forward messages via TCP, using JSON embedded in message (RFC 5424), with a
+# disk-assisted queue.
+action(
+    name="fwd-viaq"
+    type="omfwd"
+    Target="10.1.170.92"
```

Comment:
@portante, are we good with this setting for functional test?

Reply:
Ideally we'd want to use the same IP address as the jenkins executor is using.  That could be determined by looking on the executor at runtime, or we could invent another mechanism.

But I wouldn't gate this work based on this value.  For now, I'd leave this file out or comment out the action statement.

Reply:
I'll comment-out the `action` section.

However, why would we target the executor instead of targeting wherever it is that we're collecting the logs for indexing?

Reply:
We would target the executor, we want to use the same rsyslog configuration as the executor.

Reply:
See [above](https://github.com/distributed-system-analysis/pbench/pull/3260#discussion_r1113501967).